### DISCUSSION TITLE
Proxying Helm Repositories

### DISCUSSION THREAD

**Discussion by patsevanton** [goharbor/harbor#18032] [Ideas]
Hello! Thanks for harbor!
are there plans to support the helm proxy repository to access the remote repository location?

for example: https://help.sonatype.com/repomanager3/nexus-repository-administration/formats/helm-repositories#HelmRepositories-ProxyingHelmRepositories

**Comment by NicolasGoeddel** [comment]
Shouldn't this be possible already? 
I found this issue: https://github.com/goharbor/harbor/issues/6526
And this one: https://github.com/goharbor/harbor/issues/16053

You can also find "helm-hub" in the screenshot of the latest documentation: https://goharbor.io/docs/2.7.0/administration/configuring-replication/create-replication-endpoints/

But I am just trying to do it and I have no idea how to achieve it. Proxying images is no problem but how to proxy helm charts? The documentation is very unclear about that.

**Comment by WangWeiHuawwh** [comment]
收到，谢谢

**Reply by NicolasGoeddel** [reply]
> 收到，谢谢

Please answer in English.

**Reply by patsevanton** [reply]
Harbor Version v2.7.0-864aca34 not contain helm-hub

![Screenshot from 2022-12-30 20-46-34](https://user-images.githubusercontent.com/10828883/210082833-38d77697-04b1-4957-967e-3b273ebc9d3d.png)

![1672411640169](https://user-images.githubusercontent.com/10828883/210082867-13df640a-d5f2-46bd-a723-b55bc43f5786.jpg)

**Comment by patsevanton** [comment]
Any updates?

**Reply by dminca** [reply]
I can see Helm proxying in Harbor, I'm on `Version v2.5.4-a39bd2bc`

But it's only for helm.sh unfortunately
![image](https://github.com/goharbor/harbor/assets/5355219/4152c385-4f51-44fd-b0e3-a9a7df007e28)

**Reply by Vad1mo** [reply]
Since 2.8 Helm Charts are only supported in their OCI format, these you can proxy and replicate.
